libraries/rust-buildtime/context/build

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+set -e
+
+exec cargo build --release --locked
```

Comment:
it doesn't seem like you ever add this or the test script to the buildtime image? e.g. https://github.com/algorithmiaio/langpacks/blob/master/libraries/python2-buildtime/Dockerfile#L1

Reply:
Fixed in 31dbcdd